config/webpack.config.js

```diff
@@ -0,0 +1,43 @@
+const webpack = require("webpack");
```

Comment:
single quotes here please! (I'm surprised the linter didn't catch that, maybe it doesn't check for it - will open an issue)